roles/nginxplus/files/conf/http/tigerdata_staging.conf

```diff
@@ -0,0 +1,57 @@
+# Ansible managed
+proxy_cache_path /data/nginx/tigerdata-staging/NGINX_cache/ keys_zone=tigerdata-stagingcache:10m;
+
+upstream tigerdata-staging {
+    zone tigerdata-staging 64k;
+    server tigerdata-staging1.princeton.edu resolve;    
+    server tigerdata-staging2.princeton.edu resolve;
+
+    sticky learn
+          create=$upstream_cookie_pdcdescribestagingcookie
+          lookup=$cookie_pdcdescribestagingcookie
+          zone=tigerdatastagingclient_sessions:1m;
+}
+
+
+server {
+    listen 80;
+    server_name tigerdata-staging.princeton.edu;
+
+    location / {
+        return 301 https://$server_name$request_uri;
+    }
+}
+
+server {
+    listen 443 ssl http2;
+    server_name tigerdata-staging.princeton.edu;
+
+    ssl_certificate            /etc/nginx/conf.d/ssl/certs/tigerdata-staging_princeton_edu_chained.pem;
+    ssl_certificate_key        /etc/nginx/conf.d/ssl/private/tigerdata-staging_princeton_edu_priv.key;
+    ssl_session_cache          shared:SSL:1m;
+    ssl_prefer_server_ciphers  on;
+    client_max_body_size       0;
+
+    # Redirect top level traffic to /describe 
+    # until the top level has content.
+    location / {
+        return 302 https://$server_name/describe/;
+    }
+
+    location /describe/ {
```

Comment:
I don't think we want these `/describe` stanzas. I believe we want the application to run at the root. Eventually, people should go to https://tigerdata.princeton.edu and get the application. It should not redirect them to `/describe`.